### ISSUE TITLE
Fishing extravaganza

### ISSUE THREAD

**Issue by digitalfarseer** [mangosR2/scriptdev2#149]
Questgiver give quest, but players can't finish quest. Questgiver not accept quest, question mark displayed, but no complete dialog.

**Comment by Undergarun** [comment]
Db issue?

**Comment by rsa** [comment]
no, quest scripted (and script VERY old...).

**Comment by digitalfarseer** [comment]
Yep, it is broken few month ago (if not years ago). I'll post it, because it related to custom patch from this branch. :) Don't know, how fix script, DB right.
